tools/dependencies-report/src/main/resources/notices/strscan-LICENSE.txt

```diff
@@ -0,0 +1,22 @@
+Copyright (C) 1999-2006 Minero Aoki. All rights reserved.
```

Comment:
This file must be called strscan-NOTICE.txt instead of notices/strscan-LICENSE.txt:
```
The following NOTICE.txt entries are missing, please add them:
LS_HOME/tools/dependencies-report/src/main/resources/notices/strscan-NOTICE.txt
```